lib/thrift/binary/framed/client.ex

```diff
@@ -81,14 +81,15 @@ defmodule Thrift.Binary.Framed.Client do
 
      - `send_timeout`: An integer that governs how long our connection waits when sending data.
 
-    `gen_server_opts`: A keyword list of options that control the gen_server behaviour.
+     - `gen_server_opts`: A keyword list of options that control the gen_server behaviour.
 
      - `timeout`:  The amount of time to wait (in milliseconds) for a reply from a `GenServer` call.
 
   """
   @spec start_link(String.t, (0..65_535), options) :: GenServer.on_start
```

Comment:
While we do this change could we switch from `options` to `[option]`. It is no possible to compose a list type, but an element type can be composed with `[option | other_option_type]`.